deploy/cpp/include/object_detector.h

```diff
@@ -88,12 +97,13 @@ class ObjectDetector {
       const cv::Mat& raw_mat,
       std::vector<ObjectResult>* result);
 
-  std::unique_ptr<paddle::PaddlePredictor> predictor_;
+  std::shared_ptr<Predictor> predictor_;
```

Comment:
why shared_ptr?

Reply:
the change due to the upgrade of Predictor in Paddle Inference